packages/svelte/src/internal/client/dom/elements/events.js

```diff
@@ -35,14 +35,15 @@ export function replay_events(dom) {
  * @param {string} event_name
  * @param {Element} dom
  * @param {EventListener} handler
- * @param {AddEventListenerOptions} options
+ * @param {AddEventListenerOptions | boolean} options
  */
 export function create_event(event_name, dom, handler, options) {
+	const capture = typeof options === 'boolean' ? options : options.capture;
```

Comment:
Can you move this bit into the `on` method? It's only needed there, so people don't pay the cost when not using that method

Reply:
sure thing, moved via 9f09dc94618ecc86f2c4d42a63272d239cbdee30. Note my comment at https://github.com/sveltejs/svelte/pull/12030#issuecomment-2167221753.